### DISCUSSION TITLE
[Enhancement] select和select-v2组件需要一个loading插槽，有必要与其他组件效果统一

### DISCUSSION THREAD

**Discussion by zhixiaotong** [element-plus/element-plus#15527] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `N/A`

## Description

虽然是一个很简单很小的改动，但对于组件库的统一展现是有必要的。

- 之前的`autocomplete`使用的是图标加载，但未提供loading插槽。我就提了个`pr`增强效果。

- 现在的`select`和`select-v2`的加载是以文本的方式体现的，未提供`loading`插槽。

- 按理说也会影响到`TreeSelect`

是否需要类似`autocomplete`一样的提供加载插槽增强组件？

### Additional comments

_(empty)_

**Comment by zhixiaotong** [comment]
https://github.com/element-plus/element-plus/assets/34681550/4f0d472d-25d8-4714-9d7b-1c80d9eca2d7

**Comment by zhixiaotong** [comment]
@kooriookami 如果你同意这个需求，麻烦改一下，很小的改动~

**Reply by kooriookami** [reply]
合理的

**Comment by kooriookami** [comment]
已添加相关PR
